app/api/V2/utils.py

```diff
@@ -0,0 +1,32 @@
+import re
+
+
+def valid_destination_name(destination):
+    ''' validate destination name '''
```

Comment:
You should be persistence when using quotes, use either single or double quotes, don't use both.